nilearn/plotting/surf_plotting.py

```diff
@@ -12,7 +12,8 @@
 from matplotlib.cm import ScalarMappable, get_cmap
 from matplotlib.colors import Normalize, LinearSegmentedColormap
 
-from ..surface import load_surf_data, load_surf_mesh
+from ..surface import load_surf_data, load_surf_mesh, vol_to_surf, check_mesh
+from .._utils import check_niimg_3d
```

Comment:
Nilearn project is switching to absolute imports for all new & amended imports.
(We don't modify existing & unrelated to PR relative imports as that adds noise to the review process).

Reply:
Fixed